site/docs/v0.16.5/faq-new.md

```diff
@@ -0,0 +1,87 @@
+# Frequently Asked Questions
```

Comment:
I think some of your local files got pulled in; I see `faq-new.md` and `faq-newsun.md` and `faq-tmp.md`

Reply:
good catch! now you see my terrible workarounds when I switch branches and end up writing to files :stuck_out_tongue: